packages/cli/package.json

```diff
@@ -78,7 +78,7 @@
     ],
     "statements": 95,
     "branches": 92,
-    "functions": 100,
+    "functions": 94,
```

Comment:
nit - This is still above the required coverage, but maintain or increase coverage IMO

Reply:
I also removed test but those test only tested the thing that I removed